pkg/agent/cniserver/pod_configuration.go

```diff
@@ -213,12 +215,12 @@ func (pc *podConfigurator) configureInterfaces(
 
 	// Check if the OVS configurations for the container exists or not. If yes, return immediately. This check is
 	// used on Windows, as kubelet on Windows will call CNI Add for infrastructure container for multiple times
-	// to query IP of Pod. But there should be only one OVS port created for the same Pod. And if the OVS port is added more than
-	// once, OVS will return an error.
+	// to query IP of Pod. But there should be only one OVS port created for the same container. And if the OVS port is
```

Comment:
I would leave Pod here as it makes things cleared:
> But there should be only one OVS port created for the same Pod (identified by its sandbox container ID).

what do you think?

Reply:
Done